apps/accounts/views.py

```diff
@@ -161,7 +161,7 @@ def get(self, request, user, identity, next_path):
             return HttpResponseRedirect(path)
         else:
             login_user(request, user, identity)
-            next_path = next_path or '/'
+            next_path = next_path or reverse('home')
             log.info('Redirecting after login to: {}'.format(next_path))
             return HttpResponseRedirect(next_path)
```

Comment:
I know this is from an older patch but shouldn't this protect against absolute URLs? For example, it shouldn't allow `HttpResponseRedirect('http://cookie-thief.com/slurp-all-cookies.php')`. I think it only needs to do relative URLs.